### ISSUE TITLE
RegSvr32 failed with exit code 0x3

### ISSUE THREAD

**Issue by kelderek** [maphew/svg-explorer-extension#75]
When I try to install on a fresh Windows 10 Pro install, the install fails with the following message:

> C:\Program Files\SVG Explorer\Extension\SVGThumbnailExtension.dll
> 
> Unable to register the DLL/OCX: RegSvr32 failed with exit code 0x3.

I have narrowed it down to there being no VC runtime installed on the computer.  The installer tries to register the DLL before installing the VC runtimes.  If I manually install the one packaged with the installer and rerun the install, it works fine.  I have also tried with the 2015 VC runtime and it works (and then installs the 2017 VC runtime).

The installer needs to run the VC install _before_ attempting to register the DLL.

I have tried with 1.0.0 and the current dev build.  I think you could use the PrepareToInstall function of Innosetup to install it before copying the rest of the files:
https://www.jrsoftware.org/ishelp/index.php?topic=scriptevents

**Comment by kelderek** [comment]
I just created a pull request for you that resolves this issue.  I have tested it on my fresh computer and it works perfectly.  I ended up just using the existing CurStepChanged call you are using to accomplish it instead of adding a call to PrepareToInstall - they both accomplish the same goal and this keeps it simpler.  It could probably use some error checking like aborting the install if the VC runtime fails, though.
